docs/Datagrid.md

```diff
@@ -832,15 +832,16 @@ const PostList = () => (
 
 **Tip**: The `Datagrid` component `classes` can also be customized for all instances of the component with its global css name `"RaDatagrid"` as [describe here](https://marmelab.com/blog/2019/12/18/react-admin-3-1.html#theme-overrides)
 
-## Headers pinning
+## Header Pinning
 
-`<Datagrid>` pins headers by default and you don't need to do anything extra for it to work:
+`<Datagrid>` has sticky headers, which means that the header row will remain visible even when scrolling down the page.
 
 <video controls autoplay playsinline muted loop>
   <source src="./img/datagrid-headers-pinning.mp4" type="video/mp4"/>
   Your browser does not support the video tag.
 </video>
 
+You don't need to do anything for this to work, as it's enabled by default. 
```

Comment:
```suggestion
You don't need to do anything for this to work, as it's enabled by default. 

```

Reply:
I don't understand this suggestion because it seems to do the same thing as before :thinking: